src/__tests__/initializeCacheClient.test.js

```diff
@@ -0,0 +1,45 @@
+import initializeCacheClient from '../initializeCacheClient';
+
+const mockBind = jest.fn();
+const mockDisabled = jest.fn();
+const mockEnabled = jest.fn();
+
+const mockConfig = {
+    config: {
+        cache: {
+            enabled: true,
+            memcached: {
+                hosts: 'localhost:11211,localhost:11212',
```

Comment:
why both?

Reply:
why not? we do like to use tests as a usage example...